app/controllers/moirai/translation_files_controller.rb

```diff
@@ -15,12 +15,12 @@ def index
 
     def show
       @translation_keys = @file_handler.parse_file(@decoded_path)
+      @locale = @file_handler.get_first_key(@decoded_path)
+      @translations = Moirai::Translation.by_file_path(@decoded_path)
     end
```

Comment:
when we are using more than one instance variable in a controller, it smells a bit. Here we have five 😓